src/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  @for $starCount from 1 to 5 {
+    &--#{$starCount} .star:nth-child(-n + #{$starCount}) {
+      background-image: url("../images/star-active.svg");
+    }
+  }
+}
+
+.star {
```

Comment:
this is currently not part of the `stars` block

Reply:
![image](https://github.com/mate-academy/layout_catalog/assets/130931959/f5976957-b72d-4d2d-842a-1cf845dd015f)
i'm totally confused